studiopress/genesis-custom-blocks
# Add Rob's copy to the readme

#### Changes
* Applies Rob's copy

#### Testing instructions
1. `npm install --global gulp-cli` # if you haven't already
2. `npm install && gulp`
3. Expected: When copying `package/trunk/readme.txt` into the [readme validator](https://wordpress.org/plugins/developers/readme-validator/), there should only be these notices:

```
No == Upgrade Notice == section was found
No == Screenshots == section was found
No donate link was found
```
